.circleci/config.yml

```diff
@@ -13,14 +13,15 @@ jobs:
 
   dependency-check:
     docker:
-      - image: cimg/python:3.8
+      - image: cimg/python:3.7
 
```

Comment:
It looks like we are using python 3.8 in other configs - but I just logged into the running fecfile-web-api container running on cloud.gov and python3 is version Python 3.6.9 - but if you just run "python" and not "python3" you get Python 2.7.17.   Setting the Python 2/3 that aside for now, this makes me think that we should standardize everything (dev docker containers and CircleCI images) on Python 3.6.9 if that is what is really running on cloud.gov.  

Back to the Python 2 thing - Looking at our scripts, we don't specify python3, but I think it won't run on python2 at all.  I'm thinking that something is causing our app to run in python 3 on cloud.gov but I don't see where it happens. Laura, if you are doing that explicitly somewhere, can you let me know how for educational purposes?

Reply:
That's interesting! It looks like:
>The difference when you ssh in is that the profile.d scripts which setup PATH etc... are not automatically run (again, for the app runtime they are). To obtain the same environment as runtime, please run /tmp/lifecycle/shell when you first SSH in.
ref: https://github.com/cloudfoundry/python-buildpack/issues/98#issuecomment-366244014

Reply:
It looks like so far (if we go off the Dockerfile) this app is running on 3.7, and it's probably best to keep Circle in alignment with the app is running. Since this PR is already quite big, I think it might be easier to put in a separate issue to update the python version to 3.8.

Reply:
Also, using `cf ssh fecfile-web-api` and `/tmp/lifecycle/shell`, I confirmed that cloud foundry respected `runtime.txt` and is running Python 3.7.12.